### DISCUSSION TITLE
y21ga_0.3.1 on YI Home 1080 unstable image? Bricks frequently. Any suggestion?

### DISCUSSION THREAD

**Discussion by palbeda** [roleoroleo/yi-hack-Allwinner-v2#840] [Q&A] [answered]
Hi all,

I have two YI Home 1080 camera's and the migration to Allwinners v2 was easy. I did a few tests and all looked just fine, but when I wanted to install the camera's I ran into problems: after powering down the camera's in a lot of cases the camera's brick. I solved this several times with software unbrick as well as hardware unbrick as described. But each time when I power the camera's down to install them, they brick: after powering up the camera shows an orange light and does not connect to the WiFi. I do not know why the camera's brick when you power it down. It might be files that get corrupt when the camera looses its power. Unfortunately, the camera's do not know a shutdown command, so I cannot test this assumption. Does anybody have a suggestion how to get the camera's stable again?   Any help is appreciated!

Peter

**Comment by palbeda** [comment]
I am wondering if it is something completely different, like a problem caused by a 3 meter usb cable, which makes the powersupply less stable or so? I was thinking about the Wifi strenght, but that was strong enough

**Comment by roleoroleo** [comment]
Orange light means that the cam doesn't complete the boot.
It's strage.

> Unfortunately, the camera's do not know a shutdown command

You can use reboot command and unplug the power cable when the led turns off.

**Reply by palbeda** [reply, marked as answer]
Problem solved!! 

First attempt: build in a 220uF capacitor. Did not help
Second attempt: Use 2 meter USB cables of a better quality: SUCCESS!!!

Both camera's work now without any problems
